libgrust/libproc_macro/proc_macro.cc

```diff
@@ -21,3 +21,32 @@
 // <http://www.gnu.org/licenses/>.
 
 #include "proc_macro.h"
+
+namespace Procmacro {
```

Comment:
```suggestion
namespace ProcMacro {
```
nit, feel free to ignore